src/service_client/logger.py

```diff
@@ -72,30 +74,40 @@ def emit(self, record):
             self.handleError(record)
 
 
-logging.getLogger("requests").setLevel(logging.ERROR)
-logging.getLogger("urllib3").setLevel(logging.ERROR)
-
-log = logging.getLogger("")
-log.setLevel(logging.DEBUG)
-
-
-def add_log_file_handler(filename: str) -> logging.FileHandler:
-    fmt = SensitiveFormatter("%(asctime)s - %(name)s - %(levelname)s - %(thread)d - %(message)s")
-    fh = logging.FileHandler(filename)
-    fh.setFormatter(fmt)
-    log.addHandler(fh)
-    return fh
-
-
-def add_stream_handler():
-    fmt = SensitiveFormatter("%(asctime)s %(levelname)-10s - %(thread)d - %(message)s \t" "(%(pathname)s:%(lineno)d)")
-    ch = ColorizingStreamHandler(sys.stdout)
-    ch.setFormatter(fmt)
-    log.addHandler(ch)
-
-
-add_log_file_handler("test_infra.log")
-add_stream_handler()
+class Logger(ABC):
```

Comment:
what's abstract about this class?
there are no signs for anything that is abstract or that should be implemented by inheriting classes

Reply:
Just preventing from users to create instance from it

Reply:
I restored it to its previous state